ADALiOS/ADALiOS/ADOAuth2Constants.m

```diff
@@ -53,7 +53,9 @@
 NSString *const OAUTH2_SAML11_BEARER_VALUE = @"urn:ietf:params:oauth:grant-type:saml1_1-bearer";
 NSString *const OAUTH2_SAML2_BEARER_VALUE = @"urn:ietf:params:oauth:grant-type:saml2-bearer";
 
-NSString *const BROKER_PROTOCOL_VERSION         = @"broker_protocol_ver";
+NSString *const BROKER_MAX_VERSION              = @"max_protocol_ver";
```

Comment:
@RPangrle Could this be BROKER_MAX_PROTOCOL_VERSION or some such.  Right now it seems as if it is referring to the version of the broker rather than a a flavor of protocol that the broker speaks.